app/src/main/AndroidManifest.xml

```diff
@@ -47,11 +47,11 @@
             android:theme="@style/Theme.Design.Light.NoActionBar"></activity>
 
         <activity
-            android:name="com.example.mydigitalcloset.clothingUpload"
+            android:name="com.example.mydigitalcloset.clothItems.clothingUpload"
             android:exported="false"></activity>
 
         <activity
-            android:name="com.example.mydigitalcloset.clothingFront"
+            android:name="com.example.mydigitalcloset.clothItems.clothingFront"
```

Comment:
move this file out of the folder with the other page so that the navigation buttons work